documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -0,0 +1,210 @@
+$ tar xvf gcc-6.2.0.tar.gz
+$ pushd gcc-6.2.0
+$ ./configure --enable-languages=c --disable-multilib --prefix=/usr/local/gcc-6.2.0
+$ make -j5 && sudo make install
+$ popd
+
+```
+
+## Install qemu-lite
+
+Although this package installs the `qemu-lite` version of `qemu`, the binary will still be called `qemu-system-x86_64`.
+Requires at least gcc 5.
+
+```
+$ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
+$ export PATH=/usr/local/gcc-6.2.0/bin:$PATH
+$ sudo yum install libcap-ng-devel pixman-devel libattr-devel libcap-devel
+$ curl -L -O https://github.com/01org/qemu-lite/archive/3f889234fcd31ac4119aae57559b1286e09e1568.tar.gz
+$ tar xvf 3f889234fcd31ac4119aae57559b1286e09e1568.tar.gz
+$ pushd qemu-lite-3f889234fcd31ac4119aae57559b1286e09e1568
+$ CC=/usr/local/gcc-6.2.0/bin/gcc ./configure \
+                                --disable-bluez         --disable-brlapi \
+                                --disable-bzip2         --disable-curl \
+                                --disable-curses        --disable-debug-tcg \
+                                --disable-fdt           --disable-glusterfs \
+                                --disable-gtk           --disable-libiscsi \
+                                --disable-libnfs        --disable-libssh2 \
+                                --disable-libusb        --disable-linux-aio \
+                                --disable-lzo           --disable-opengl \
+                                --disable-rbd           --disable-qom-cast-debug \
+                                --disable-rdma          --disable-sdl \
+                                --disable-seccomp       --disable-slirp \
+                                --disable-snappy        --disable-spice \
+                                --disable-strip         --disable-tcg-interpreter \
+                                --disable-tcmalloc      --disable-tools \
+                                --disable-tpm           --disable-usb-redir \
+                                --disable-uuid          --disable-vnc \
+                                --disable-vte           --disable-vnc-{jpeg,png,sasl} \
+                                --disable-xen           --enable-attr \
+                                --enable-cap-ng         --enable-kvm \
+                                --enable-virtfs         --target-list=x86_64-softmmu \
+                                --extra-cflags="-fno-semantic-interposition -O3 -falign-functions=32" \
+                                --datadir=/usr/local/share/qemu-lite \
+                                --libdir=/usr/local/lib64/qemu-lite \
+                                --libexecdir=/usr/local/libexec/qemu-lite
+$ make -j5 && sudo make install
+$ popd
+```
+
+## Install container kernel and rootfs
+
+- Point your browser at the URL below and download the latest versions of `linux-container` and `clear-containers-image` packages:
```

Comment:
I would suggest to use that repo to install the dependencies and avoid the user to look for a particular number version etc, this will allow to automate the process latter. 

```
# yum-config-manager --add-repo http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/CentOS_7/home:clearlinux:preview:clear-containers-2.1.repo 
# yum update
# yum install linux-container clear-containers-image

```